apps/explorer/lib/contracts.ts

```diff
@@ -95,6 +95,7 @@ export function normalizeContract(
       filesize: contract.v2FileContract.filesize,
       hostBurned: undefined,
       hostPayoutMissed: {
+        id: contract.v2FileContract.hostOutput.id,
```

Comment:
Contract outputs do not currently have an ID field

Reply:
Ah ok since its option I can remove this anyway.